__tests__/persistent-storage.spec.ts

```diff
@@ -25,33 +25,21 @@ it("returns default when schema is not present", async () => {
   expect(state).toEqual(defaultPersistentState)
 })
 
-import { defaultTheme } from "@app/theme/default-theme"
-
-it("migration from 4 to 5", async () => {
-  const state4 = {
-    schemaVersion: 4,
-    hasShownStableSatsWelcome: false,
-    isUsdDisabled: false,
-    galoyInstance: {
-      id: "Main",
-      name: "BBW",
-      graphqlUri: "https://api.mainnet.galoy.io/graphql",
-      graphqlWsUri: "wss://api.mainnet.galoy.io/graphql",
-      posUrl: "https://pay.bbw.sv",
-      lnAddressHostname: "pay.bbw.sv",
-    },
+it("migration from 5 to 6", async () => {
+  const state5 = {
+    schemaVersion: 5,
+    galoyInstance: { id: "Main" },
     galoyAuthToken: "myToken",
-    isAnalyticsEnabled: true,
-    theme: defaultTheme,
   }
 
-  const state5 = {
-    schemaVersion: 5,
+  const state6 = {
+    schemaVersion: 6,
     galoyInstance: { id: "Main" },
     galoyAuthToken: "myToken",
+    isAuthenticatedWithDeviceAccount: false,
```

Comment:
still have to read the rest of the PR, but I'm wondering if state is needed. having less state on persistant storage is imo better, so if we could infer this from other way I think that is better.

Reply:
I dont think we can infer it. We dont save a token with the device account because we need it to be constantly refreshed (we use the pattern provided in the documentation).